tests/raftstore/cluster.rs

```diff
@@ -458,6 +459,43 @@ impl<T: Simulator> Cluster<T> {
         assert!(status_resp.has_region_detail());
         status_resp.take_region_detail()
     }
+
+    // NOTE: if you have set transport hooks before, call this function will overwrite them
+    pub fn partition(&mut self, s1: Arc<HashSet<u64>>, s2: Arc<HashSet<u64>>) {
+        for node_id in s1.as_ref() {
```

Comment:
compiler will complain

        error: the trait bound `&std::sync::Arc<std::collections::HashSet<u64>>: std::iter::Iterator` is not satisfied [E0277]